pipeline/src/types/transformed/eventDocument.ts

```diff
@@ -0,0 +1,30 @@
+import { Image } from ".";
+
+export type EventDocument = {
+  type: "Event";
+  id: string;
+  title: string;
+  image?: Image;
+  times: { startDateTime: Date | null; endDateTime: Date | null }[];
```

Comment:
Can we do the same here or does it have to be `null`?

Reply:
Aligning on how the FE handles this, it is now optional to account for the possibility of it being undefined (instead of null)